### ISSUE TITLE
Replace use of xlrd with openpyxl

### ISSUE THREAD

**Issue by dgdekoning** [brightway-lca/brightway2-io#86]
As noted on the [ReadTheDocs](https://xlrd.readthedocs.io/en/latest/?badge=latest) page, `xlrd` will no longer support the reading of .xlsx files as of version 2.0 (shipped the 11th of december).

In response, the newer version of pandas have switched from [using xlrd to openpyxl](https://github.com/pandas-dev/pandas/issues/38424).

I would suggest going with the flow and swapping out excel-reading packages. As the current implementation causes `create_default_lcia_methods()` to fail when the newest version of xlrd is installed.

```bash
 File "C:\Users\me\workspace\activity-browser\activity_browser\app\ui\wizards\db_import_wizard.py", line 1215, in run
    bw.create_default_lcia_methods()
  File "C:\Users\me\AppData\Local\Continuum\miniconda3\envs\ab_test\lib\site-packages\bw2io\__init__.py", line 91, in create_default_lcia_methods
    ei = EcoinventLCIAImporter()
  File "C:\Users\me\AppData\Local\Continuum\miniconda3\envs\ab_test\lib\site-packages\bw2io\importers\ecoinvent_lcia.py", line 30, in __init__
    self.csv_data, self.cf_data, self.units, self.file = convert_lcia_methods_data()
  File "C:\Users\me\AppData\Local\Continuum\miniconda3\envs\ab_test\lib\site-packages\bw2io\data\__init__.py", line 338, in convert_lcia_methods_data
    "CFs"
  File "C:\Users\me\AppData\Local\Continuum\miniconda3\envs\ab_test\lib\site-packages\bw2io\data\__init__.py", line 40, in get_sheet
    wb = xlrd.open_workbook(path)
  File "C:\Users\me\AppData\Local\Continuum\miniconda3\envs\ab_test\lib\site-packages\xlrd\__init__.py", line 170, in open_workbook
    raise XLRDError(FILE_FORMAT_DESCRIPTIONS[file_format]+'; not supported')
xlrd.biffh.XLRDError: Excel xlsx file; not supported
```

Relevant code:

https://github.com/brightway-lca/brightway2-io/blob/388e2ea9722a123f7e192250c21c2996e8e024cd/bw2io/data/__init__.py#L35-L37

**Comment by tngTUDOR** [comment]
A quick ugly fix would be to pin the version of xlrd to <= 2.0.0
https://xlrd.readthedocs.io/en/latest/changes.html#id1

**Comment by tngTUDOR** [comment]
For those waiting for a new version of the package that fixes this, you can use conda or pip to downgrade the xlrd package with:

```
conda install xlrd<2.0.0
```

or

```
pip install xrd<2.0.0
```

**Comment by tngTUDOR** [comment]
xlrd is used in the following places:

```
extractors/excel.py
3:import xlrd
18:        wb = xlrd.open_workbook(filepath)
```

```
extractors/exiobase.py
5:import xlrd
24:        wb = xlrd.open_workbook(os.path.join(path, "types_version2.2.2.xls"))
```

```
data/__init__.py
17:import xlrd
36:    wb = xlrd.open_workbook(path)
```

**Comment by cmutel** [comment]
Fixed in 0.8 release.
